drivers/i2c/i2c_ite_it8xxx2.c

```diff
@@ -72,6 +84,8 @@ struct i2c_it8xxx2_data {
 	uint8_t freq;
 	/* wait for stop bit interrupt */
 	uint8_t stop;
+	/* Number of messages. */
+	uint8_t num_msgs;
```

Comment:
You only use this with FIFO mode, so please guard with CONFIG_I2C_IT8XXX2_FIFO_MODE.

Reply:
Done.